libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt

```diff
@@ -241,85 +282,40 @@ object ApolloCompiler {
         fragmentDefinitions = fragments,
         allFragmentDefinitions = allFragmentDefinitions,
         codegenModels = codegenModels,
-        generateOptionalOperationVariables = options.generateOptionalOperationVariables,
-        flattenModels = options.flattenModels,
-        decapitalizeFields = options.decapitalizeFields,
-        alwaysGenerateTypesMatching = options.alwaysGenerateTypesMatching,
-        generateDataBuilders = options.codegenSchema.generateDataBuilders,
+        generateOptionalOperationVariables = options.generateOptionalOperationVariables ?: defaultGenerateOptionalOperationVariables,
+        flattenModels = flattenModels,
+        decapitalizeFields = options.decapitalizeFields ?: defaultDecapitalizeFields,
+        alwaysGenerateTypesMatching = options.alwaysGenerateTypesMatching ?: defaultAlwaysGenerateTypesMatching,
+        generateDataBuilders = codegenSchema.generateDataBuilders,
         fragmentVariableUsages = validationResult.fragmentVariableUsages
     ).build()
   }
 
-  fun buildUsedCoordinates(irOperations: IrOperations): UsedCoordinates {
-    return irOperations.usedFields
-  }
-
-  fun buildUsedCoordinates(irOperationsFiles: List<File>): UsedCoordinates {
-    val irOperations = irOperationsFiles.map { it.toIrOperations() }
-
-    val duplicates = irOperations.flatMapIndexed { index, irOperations1 ->
-      irOperations1.fragmentDefinitions.map { it.name to irOperationsFiles.get(index).path }
-    }.groupBy { it.first }.mapValues { it.value.map { it.second } }
-        .entries
-        .filter { it.value.size > 1 }
-        .flatMap { entry ->
-          entry.value.map {
-            "- ${entry.key}: $it"
-          }
-        }
```

Comment:
Turns out this was not really needed as the check is already done [here](https://github.com/apollographql/apollo-kotlin/blob/7c2c8eeb6421718bfa2ada8c7e025d86d8577977/libraries/apollo-ast/src/commonMain/kotlin/com/apollographql/apollo3/ast/internal/ExecutableValidationScope.kt#L760). 

This means it's ok to have clashing fragments in sibling modules as long as they have different package names.